src/patternfly/components/Check/check.scss

```diff
@@ -6,13 +6,15 @@
   --pf-c-check__label--FontSize: var(--pf-global--FontSize--md);
   --pf-c-check__label--LineHeight: var(--pf-global--LineHeight--sm);
   --pf-c-check__input--MarginTop: #{pf-size-prem(-3px)};
+  --pf-c-check__input--Height: calc(var(--pf-c-check__label--FontSize) * var(--pf-c-check__label--LineHeight));
+  --pf-c-check__input--MarginTop: calc(-1px * (var(--pf-c-check__label--LineHeight) * 1.25));
```

Comment:
is there any significance to 1.25 here or is that just the number that worked the best?

Reply:
Excellent question. There appears to be a discrepancy in how FF, Safari and Chrome calculate `line-height` + `font-size`. Chrome & Safari calculate `height` (`1.3 * 16px`) as `20px`, or `_floor`, whereas FF calculates `height` as `20.8px`, the actual value. Bumping the negative margin by `.25` nudges the input up slightly to accomodate the FF difference.